common/src/WooCommerce.php

```diff
@@ -301,9 +309,7 @@ public function syncReviews() {
             return;
         }
         $this->processReviews($reviews);
-        if ($last_modified = (string) ($reviews[0]->modified ?? null)) {
-            update_option($this->plugin->getOptionName('last_synced'), $last_modified);
-        }
+        update_option($this->plugin->getOptionName('last_synced'), date(\DateTime::RFC3339));
```

Comment:
No. This will miss reviews that are created between the start and end of the sync.